src/js/utils/dom.js

```diff
@@ -329,7 +328,7 @@ export function findElPosition(el) {
 
   // Android sometimes returns slightly off decimal values, so need to round
   return {
-    left: roundFloat(left),
```

Comment:
I think these should these just be Math.round() now. Back when `roundFloat` was just `round` it was used for most rounding.